app/app.tsx

```diff
@@ -86,6 +91,28 @@ function App() {
     }
   }, [rootStore, navigationRef])
 
+  useEffect(() => {
+    const listener = (item: ShortcutItem) => {
+      const origin = stations.find((station) => station.id === item.data.originId)
+      const destination = stations.find((station) => station.id === item.data.destinationId)
```

Comment:
I wouldn't want to override the existing user route, seems counter intuitive to me - but if that requires a big refactor let's leave it like that.

Reply:
I think we'll need a refactor because `routePlan` store is the single source of truth here, but also the current behavior is desired. Just like selecting a favorite route from the select-station-view, just faster and directly from the home screen.